backends/blocked/ceed-blocked-operator.c

```diff
@@ -16,9 +16,9 @@
 //------------------------------------------------------------------------------
 // Setup Input/Output Fields
 //------------------------------------------------------------------------------
-static int CeedOperatorSetupFields_Blocked(CeedQFunction qf, CeedOperator op, bool is_input, bool *skip_rstr, const CeedInt block_size,
-                                           CeedElemRestriction *block_rstr, CeedVector *e_vecs_full, CeedVector *e_vecs, CeedVector *q_vecs,
-                                           CeedInt start_e, CeedInt num_fields, CeedInt Q) {
+static int CeedOperatorSetupFields_Blocked(CeedQFunction qf, CeedOperator op, bool is_input, bool *skip_rstr, CeedInt *e_data_out_indices,
+                                           bool *apply_add_basis, const CeedInt block_size, CeedElemRestriction *block_rstr, CeedVector *e_vecs_full,
```

Comment:
and by "similar-ish", I'm of course referring to a [comment in a completely different PR](https://github.com/CEED/libCEED/pull/1637#discussion_r1724268282)... :sweat_smile:

Reply:
As you likely know, these forms have identical meaning to the compiler so it's entirely (unchecked) documentation to write it as `bool a[]` versus `bool *a`. One reason to prefer `[]` is that it may be shown that way by your editor's language server/autocomplete. I don't have strong feelings, but it's out of scope of this PR.

Reply:
We mostly have `bool *a` over `bool a[]` internally, but we can pick one and standardize as a follow-up